schema.yaml

```diff
@@ -99,6 +99,16 @@
         visible: true
         default: false
 
+      pg_whitelisted_ip: 
+        type: string
+        visible: 
+           not:
+             - create_in_private_subnet
+        required: false
+        title: "Whitelisted IP"
+        description: "Choose whitelisted IP in public scenario."
```

Comment:
Replace with 
`description: "Choose whitelisted IP in a public scenario (for example, 89.64.91.8). It will give you the ability to access the database from your laptop and PGAdmin tool."`